ddtrace/tracer/tracer.go

```diff
@@ -329,6 +331,14 @@ func newTracer(opts ...StartOption) *tracer {
 			t.reportRuntimeMetrics(defaultMetricsReportInterval)
 		}()
 	}
+	if c.runtimeMetricsV2 {
+		l := slog.New(slogHandler{})
```

Comment:
Why use slog instead of tracer logger? Do these logs get routed somewhere else that customers don't see / only we see somewhere?